app.js

```diff
@@ -0,0 +1,252 @@
+    'Experiences of a guitar playing DataStax developer',
+    ['Franky J'],
+    '42 min',
+    ['Action'],
+    'TV-18',
+    2020,
+    'Movie']
+
+  const paramsPulp = [TITLE_PULP_FICTION, SHOW_ID_PULP_FICTION,
+    ['John Travolta', 'Samuel L. Jackson',
+      'Uma Thurman', 'Harvey Keitel', 'Tim Roth', 'Amanda Plummer', 'Maria de Medeiros',
+      'Ving Rhames', 'Eric Stoltz', 'Rosanna Arquette', 'Christopher Walken',
+      'Bruce Willis'],
+    ['United States'],
+    new Date(2019, 1, 19),
+    'This stylized crime caper weaves together stories ...',
+    ['Quentin Tarantino'],
+    '42 min',
+    ['Classic Movies', 'Cult Movies', 'Dramas'],
+    'R',
+    1994,
+    'Movie']
+
+  console.log("paramsJimmy: " + paramsJimmy)
+  console.log('Inserting into Primary Table for title: %s', TITLE_LIFE_OF_JIMMY)
+  await client.execute(insertPrimary, paramsJimmy, { prepare: true })
+
+  console.log('Inserting into Primary Table for title: %s', TITLE_PULP_FICTION)
+  await client.execute(insertPrimary, paramsPulp, { prepare: true })
+}
+
+async function insertTitlesByDateRecords() {
+
+  const insertTitlesByDate = 'INSERT INTO ' +
+    TABLE_NETFLIX_TITLES_BY_DATE +
+    ' (show_id, date_added, release_year, title, type) VALUES (?,?,?,?,?) IF NOT EXISTS;'
+
+  const paramsJimmy = [
+    SHOW_ID_LIFE_OF_JIMMY,
+    new Date(2020, 6, 1),
+    2020,
+    TITLE_LIFE_OF_JIMMY,
+    'Movie'
+  ]
+
+  const paramsPulp = [
+    SHOW_ID_PULP_FICTION,
+    new Date(2019, 1, 19),
+    1994,
+    TITLE_PULP_FICTION,
+    'Movie'
+  ]
+
+  console.log('Inserting into TitlesByDate Table for title: %s', TITLE_LIFE_OF_JIMMY)
+  await client.execute(insertTitlesByDate, paramsJimmy, { prepare: true })
+
+  console.log('Inserting into TitlesByDate Table for title: %s', TITLE_PULP_FICTION);
+  await client.execute(insertTitlesByDate, paramsPulp, { prepare: true })
+}
+
+async function insertTitlesByRatingRecords() {
+
+  const insertTitlesByRating = 'INSERT INTO ' +
+    TABLE_NETFLIX_TITLES_BY_RATING +
+    ' (show_id, rating, title) VALUES (?,?,?) IF NOT EXISTS;'
+
+  const paramsJimmy = [
+    SHOW_ID_LIFE_OF_JIMMY,
+    "TV-18",
+    TITLE_LIFE_OF_JIMMY
+  ];
+
+
+  const paramsPulp = [
+    SHOW_ID_PULP_FICTION,
+    "R",
+    TITLE_PULP_FICTION
+  ]
+
+  console.log('Inserting into TitlesByRating Table for title: %s', TITLE_LIFE_OF_JIMMY)
+  await client.execute(insertTitlesByRating, paramsJimmy, { prepare: true })
+
+  console.log('Inserting into TitlesByRating Table for title: %s', TITLE_PULP_FICTION)
+  await client.execute(insertTitlesByRating, paramsPulp, { prepare: true })
+}
+
+async function readAll(tableName) {
+
+  const selectAll = 'SELECT * FROM ' + tableName + ' ALLOW FILTERING;'
+  console.log('Selecting all from Table: %s', tableName)
+  return client.execute(selectAll)
+
+}
+
+async function readAllInPrimaryByTitle(titleName) {
+
+  const selectAll = 'SELECT * FROM ' + TABLE_NETFLIX_PRIMARY + ' WHERE title = ? ALLOW FILTERING;'
```

Comment:
shouldn't need ALLOW FILTERING here and we definitely don't want to showcase that to the user